src/components/topic/style/topic.less

```diff
@@ -80,14 +80,50 @@
     background-position: -110px -240px;
   }
 
-  .ga-topics-sprite-dev {
+  /*.ga-topics-sprite-dev {
     background-position: -220px -240px;
-  }
+  }*/
 
   .ga-topics-sprite-emapis {
     background-position: -220px -240px;
   }
 
+  .ga-topics-sprite-dev {
+    width: 110px;
+    height: 60px;
+    background-position: 0 -300px;
+    -webkit-animation: play 2.5s steps(28) infinite;
+       -moz-animation: play 2.5s steps(28) infinite; 
+        -ms-animation: play 2.5s steps(28) infinite;
+         -o-animation: play 2.5s steps(28) infinite;
+            animation: play 2.5s steps(28) infinite;
+  } 
+   
+  @-webkit-keyframes play {
+    from { background-position:    0 -300px; }
+      to { background-position: -440px -300px; }
+  }
+
+  @-moz-keyframes play {
+    from { background-position:    0 -300px; }
+      to { background-position: -440px -300px; }
+  }
+
+  @-ms-keyframes play {
+    from { background-position:    0 -300px; }
+      to { background-position: -440px -300px; }
+  }
+
+  @-o-keyframes play {
+    from { background-position:    0 -300px; }
+      to { background-position: -440px -300px; }
+  }
+
+  @keyframes play {
+    from { background-position:    0 -300px; }
+      to { background-position: -3080px -300px; }
+  }
+
```

Comment:
Use also a mixin:
```
.play-frames () {
	from { background-position:    0 -300px; }
      to { background-position: -440px -300px; }
}
 @-webkit-keyframes play {.play-frames;}
 @keyframes play {.play-frames;}
```
No need of ms, o,and moz prefixed, see http://caniuse.com/#search=keyframes